armory/datasets/config_load.py

```diff
@@ -0,0 +1,85 @@
+"""
+Temporary file for testing loading from config without modifying armory.utils
+"""
+
+import copy
+
+from armory.datasets import load, preprocessing, generator
+
+
+def actual_loader(
+    name,
+    version=None,
+    split="test",
+    shuffle_files=False,
+    framework="numpy",
+    epochs=1,
+    drop_remainder=False,
+    num_batches=None,
+    batch_size=1,
+    shuffle_elements=False,
+):
+    info, ds_dict = load.load(name, version=version, shuffle_files=shuffle_files)
+    preprocessor = preprocessing.get(name, version=version)
+    return generator.ArmoryDataGenerator(
+        info,
+        ds_dict,
+        split=split,
+        batch_size=batch_size,
+        framework=framework,
+        epochs=epochs,
+        element_filter=None,
+        element_map=preprocessor,
+        shuffle_elements=shuffle_elements,
+    )
+
+
+def load_dataset(dataset_config, *args, check_run=False, **kwargs):
```

Comment:
what `args` should `load_dataset()` be receiving beyond `dataset_config`?

Reply:
It's really weird. But it comes from scenarios passing in things. I don't think args is currently used, but this is the function signature for the existing `armory.utils.config_loading.load_dataset` method, so I duplicated that here.